tuto2/src/demo.ml4

```diff
@@ -0,0 +1,356 @@
+(* This will be compilable once the following pull-requests:
+   - https://github.com/coq/coq/pull/759
+   - https://github.com/coq/coq/pull/761
+   will be merged.
+
```

Comment:
Can you remove the previous four lines: the pull request were merged AFAICT

Reply:
Ok. See commit a7bf179.